libs/auth/src/card_reader.test.ts

```diff
@@ -132,6 +132,7 @@ test('CardReader status changes', () => {
   newCardReader();
 
   mockPcscLite.emit('error');
+  expect(onReaderStatusChange).toHaveBeenCalledTimes(1);
```

Comment:
This kind of explicit enumeration of expected calls is what `test_utils/mock_function` is designed for. No need to change but just mentioning it

Reply:
For sure! That did cross my mind